### ISSUE TITLE
Dualshock 3 doesn't pair in 3.5.1

### ISSUE THREAD

**Issue by bennasar99** [ValveSoftware/SteamOS#1180]
#### Your system information

* Steam client version: Latest beta in 22/10/2023
* SteamOS version: 3.5.1
* Opted into Steam client beta?: [Yes/No]  Yes
* Opted into SteamOS beta?: [Yes/No]  Yes
* Have you checked for updates in Settings > System?: [Yes/No]

#### Please describe your issue in as much detail as possible:
I conmectes my dualshock 3 via usb, expecting to disconnect it after a few seconds and then play wirelessly. It didn't happen, I could only play with it plugged in.
Rollback to stable steamos (3.4.11) fixed the issue.

#### Steps for reproducing this issue:

1. Plug Dualshock 3 via usb.
2. Wait a few seconds.
3. Unplug the controller.  It will not pair.

**Comment by Tau5** [comment]
Can replicate on 3.5.5 stable.
Rolling back to 3.4.11 stable fixes the issue

**Comment by Tau5** [comment]
I tested it on desktop mode before going back to 3.4.11 and it seems that when opening bluetoothctl and connecting the controlller, nothing happens.

[According to the Arch Wiki](https://wiki.archlinux.org/title/Gamepad#Pairing_via_Bluetooth) and on my experience, when connecting the controller, bluetoothctl should show a prompt to authorise and pair the controller. This doesn't happen on 3.5.5

This is probably because `bluez-plugins` may be missing from 3.5.5, though I can't check it as I'm now on 3.4.11

**Comment by Tau5** [comment]
More information:

`bluez-plugins` is present on 3.5.5 and a Dualshock 3 controller may connect via bluetooth IF it is already paired (for example, from another update) but it cannot be paired on 3.5.5

**Comment by bennasar99** [comment]
Do you know if there's a way to pair it "externally"? Like getting the Bluetooth adresses of the steam deck and using a PC to link DS3 to that adress.

I would like to use my ds3 on my soon to arrive OLED, which probably comes with 3.5.

**Comment by Tau5** [comment]
@bennasar99 No, but maybe if you boot a live linux distro (not windows) on the deck and pair the controller in there it will be paired (that's basically what you would be doing if you pair it in 3.4.11 and use it on 3.5.7)

**Comment by adrenaliner** [comment]
There is a workaround:

1. Start Desktop
2. Start Terminal
3. bluetoothctl devices Trusted
4. bluetoothctl remove macadress to remove the old connection to the Controller
5. Reboot and connect through USB again and unplug again

Just found it on Steam forums, tested it and worked, thought I should share.
